src/features/processPicker.ts

```diff
@@ -45,6 +46,27 @@ export class RemoteAttachPicker {
 
     private static _channel: vscode.OutputChannel = null;
 
+    public static ValidateAndFixPipeProgram(program: string): Promise<string> {
+        return PlatformInformation.GetCurrent().then(platformInfo => {
+            // If its 64 bit windows and the program does not exist. Try to replace System32 with sysnative
+            if (platformInfo.isWindows && platformInfo.architecture === "x86_64" && !fs.existsSync(program) && program.toLowerCase().includes("system32")) {
```

Comment:
It would be nice if we don't enter the if block if the following condition is not satisfied program.toLowerCase().startsWith(process.env.SystemRoot + "system32").